Packs/CommonScripts/ReleaseNotes/1_1_0.md

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
If you are changing the version in the pack metadata you should probably change the name of this file too